content/lang/de/guide/differences-to-react.md

```diff
@@ -50,10 +50,10 @@ Preact fügt durchaus vereinzelt nützliche Funktionen hinzu, die von dem Schaff
 ## Was fehlt?
 
 - [PropType] Validierung: Nicht jeder benutzt PropTypes, daher gehören sie nicht Preacts Kern an.
-    - _**PropTypes sind vollständig durch** [preact-compat] **unterstützt**, können aber auchmaniell verwendet werden._
+    - _**PropTypes sind vollständig durch** [preact-compat] **unterstützt**, können aber auch manuell verwendet werden._
 - [Children]: Nicht notwendig in Preact, da `props.children` _immer ein Array_ ist.
     - _`React.Children` ist vollständig in [preact-compat] unterstützt._
-- Synthetisch Ereignisse: Preacts Ziel in der Browserunterstützung setzt diesen zusätzlichen Mehraufwand nicht vorraus.
+- Synthetische Ereignisse: Preacts Ziel in der Browserunterstützung setzt diesen zusätzlichen Mehraufwand nicht vorraus.
```

Comment:
Sounds strange translating `events` to `Ereignisse`. It is the correct word in german but we usually do not translate technical terms. The german mdn docs don't translate `events` either: https://developer.mozilla.org/de/docs/Web/API/EventListener